File: Selenium/Project/CRMActivity4.java
package examples;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.WebDriverWait;

import java.time.Duration;

public class CRMActivity4 {

    public static void main(String[] args)
    {
        WebDriverManager.firefoxdriver().setup();

        WebDriver driver = new FirefoxDriver();
        driver.get("http://alchemy.hguy.co/crm");
        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(50));
        driver.findElement(By.id("user_name")).sendKeys("admin");
        driver.findElement(By.id("username_password")).sendKeys("pa$$w0rd");
        driver.findElement(By.id("bigbutton")).click();
        //verifying the homepage is displayed or not
        if(driver.findElement(By.xpath("//a[text()='Sales']")).isDisplayed()){
            System.out.println("Home Page is displayed");
        }else{
            System.out.println("Home Page is not displayed");
        }
        driver.close();


    }
}

File: Selenium/Project/CRM_Activity4.py
# Logging into the site
# Goal: Open the site and login with the credentials provided

# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By

# Start the Driver
with webdriver.Chrome() as driver:

    driver.get("https://alchemy.hguy.co/crm/")

# Find and select the username and password fields
# Enter login credentials into the respective fields
    driver.find_element(By.ID,"user_name").send_keys("admin")
    driver.find_element(By.ID,"username_password").send_keys("pa$$w0rd")
    driver.find_element(By.ID,"bigbutton").click()
    homePageHeader = driver.find_element(By.ID,"tab0").text
    # print(homePageHeader)
if(homePageHeader=="SUITECRM DASHBOARD"):
        print("Home Page Opened Successfully !")
else:
        print("Login failed !")
# Close the browser
driver.quit()